scripts/create_summary.ts

```diff
@@ -143,7 +143,7 @@ async function createSummary(
     ...licenseIssues.unlicensed
   ]
 
-  summary.addScannedDependencies(allChanges)
+  summary.addScannedFiles(allChanges)
```

Comment:
this still doesn't totally protect against the overflow scenario, but yes, much less likely that there would be 1MB worth of filechanges

Reply:
@jonjanego that's right, we could also do a check against the overflow scenario (1048576 bytes) and trim the summary if so. I omitted it because, as you say, without listing the individual dependency changes it's exceedingly unlikely the summary would go over 1MB.

Reply:
if it is not _too much_ effort, it may be good to do that just to assure ourselves of not hitting this failure again.  I started to take a crack at it, but backed away due to time constraints... but maybe there is a simpler way than how i was looking at it.

Reply:
Updated the PR to handle this too ✅ ready for review 👀